### PR TITLE
[LLVM] fix compilation on macOS

### PR DESCRIPTION
**Pull request by efeldblum-redis**

### CODE REVIEW

File: src/trie/trie.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include <stdbool.h>
 #include "rune_util.h"
 #include "redisearch.h"
+#include "util/timeout.h"
```
[the remark this answers is not in the document]
Comment (oshadmi) [reply src/trie/trie.h:10]: If you keep in trie.c  the `#include "util/timeout.h"`
And in trie.h you only needs to know 
```
void TrieNode_IterateContains(TrieNode *n, const rune *str, int nstr, bool prefix, bool suffix,
                              TrieRangeCallback callback, void *ctx, struct timespec *timeout);
```

So I though that fwd decl could suffice.

[the remark this answers is not in the document]
Comment (efeldblum-redis) [reply src/trie/trie.h:10]: Yes, I concur.

**Review by oshadmi — APPROVED** [review]
👍

**Review by ashtul — COMMENTED** [review]
@efeldblum-redis Welcome to the team :1st_place_medal:

**Pull request merged**